### PR TITLE
Adding the verification name to the aggregator job

### PR DESCRIPTION
**Pull request by bradmwilliams**

### CODE REVIEW

File: cmd/release-controller/sync_verify_prow.go [hunk 1]
```diff
@@ -62,6 +62,9 @@ func (c *Controller) ensureProwJobForReleaseTag(release *Release, verifyName str
 		return nil, terminalError{err}
 	}
 
+	if isAggregatedJob {
+		periodicConfig.Name = fmt.Sprintf("%s-%s", jobName, verifyName)
```
Comment (deads2k) [inline cmd/release-controller/sync_verify_prow.go:66]: can this be moved to the lower isAggregatedJob block?

Comment (deads2k) [inline cmd/release-controller/sync_verify_prow.go:66]: where do periodicConfigs come from?  can you link to an example config.

Comment (bradmwilliams) [reply cmd/release-controller/sync_verify_prow.go:66]: Unfortunately no.  It *must* happen after fetching and validating the `periodicConfig`.

[the remark this answers is not in the document]
Comment (bradmwilliams) [reply cmd/release-controller/sync_verify_prow.go:66]: No.  This is the "hacky" part of this PR...

Comment (bradmwilliams) [reply cmd/release-controller/sync_verify_prow.go:66]: This is the `periodicConfig` for the aggregator job itself:
https://github.com/openshift/release/blob/23bd2851d5634cbd7912d1d4578d714881894f6f/ci-operator/jobs/openshift/release/openshift-release-infra-periodics.yaml#L1540-L1624

Comment (deads2k) [reply cmd/release-controller/sync_verify_prow.go:66]: can't be done near line 74>

Comment (deads2k) [reply cmd/release-controller/sync_verify_prow.go:66]: How do the upgrade jobs get their periodicConfig.Name which is distinct for each upgrade version without a separate entry in periodics.yaml?

Comment (bradmwilliams) [reply cmd/release-controller/sync_verify_prow.go:66]: At line 68, the `periodicConfig` is converted over to `ProwJobSpec`, that sets the name from then on.

Comment (bradmwilliams) [reply cmd/release-controller/sync_verify_prow.go:66]: Smoke and mirrors... or the release-config:
https://github.com/openshift/release/blob/aaa3353f6f9ad4d2738ae48648744b150dda70c3/core-services/release-controller/_releases/release-ocp-4.10-ci.json#L76

Comment (deads2k) [reply cmd/release-controller/sync_verify_prow.go:66]: Ok, I'm finally getting there.

Can I get it as `verifyName-jobName`?  If I understand it correctly, that will let me name my verify stanzas as `aggregated-gcp-micro-upgrade` and I'll get a name `aggregated-gcp-micro-upgrade-periodic-ci-openshift-release-master-ci-4.10-e2e-gcp-upgrade`

Comment (bradmwilliams) [reply cmd/release-controller/sync_verify_prow.go:66]: As currently written, the aggregator job name is actually: `release-openshift-release-analysis-aggregator`
So, adding the `verifyName` as a prefix or suffix would actually have ^^ in it's name:
 aggregated-gcp-micro-upgrade-release-openshift-release-analysis-aggregator
or
 release-openshift-release-analysis-aggregator-aggregated-gcp-micro-upgrade

Comment (bradmwilliams) [reply cmd/release-controller/sync_verify_prow.go:66]: That being said, I have access to the prowJob name if you'd prefer the name to be something like you've suggested?

**Pull request merged**